qiskit/transpiler/passes/synthesis/hls_plugins.py

```diff
@@ -1190,7 +1190,7 @@ class MCXSynthesis2CleanKG24(HighLevelSynthesisPlugin):
 
     For a multi-controlled X gate with :math:`k\ge 3` control qubits this synthesis method requires
     :math:`2` additional clean ancillary qubits. The synthesized circuit consists of :math:`k + 2`
-    qubits and at most :math:`12 * k - 18` CX gates.
+    qubits and at most :math:`6 * k - 6` CX gates.
```

Comment:
The synthesized circuit consists of k + 3 qubits, right? (k controls, 1 target and 2 ancillas)

Reply:
Yes, fixed.